docs/mailchimp-integration.md

```diff
@@ -25,7 +25,7 @@ mailchimp.set_config(
         "server": settings.MAILCHIMP_API_KEY.split("-")[-1],
     }
 )
-mailchimp.get_list_interest_categories(MAILCHIMP_LIST_ID)
+mailchimp.lists.get_list_interest_categories(MAILCHIMP_LIST_ID)
```

Comment:
Not entirely, I noticed I didn't write this command properly before when setting up mailchimp for testing, figured it would be best to correct it here.